### ISSUE TITLE
Backup copies?

### ISSUE THREAD

**Issue by j-fu** [fonsp/Pluto.jl#436]
Hi, is it possible to introduce a backup mechanism which saves notebooks before overwriting them ?

**Comment by fonsp** [comment]
Yeah this would be cool! I currently use the .jl export (top right) for this, but it's kind of a hassle.

**Comment by fonsp** [comment]
@dralletje
